src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -91,6 +92,12 @@
   @Override
   public CacheHandle lookup(Spawn spawn, SpawnExecutionContext context)
       throws InterruptedException, IOException, ExecException {
+    boolean checkCache = options.remoteAcceptCached && Spawns.mayBeCached(spawn);
```

Comment:
This is just lifting the condition from line 119, so I can reuse it for deciding whether to report or not.

The block below was reindented since it doesn't make sense to execute any of that code if the cache isn't going to be consulted.